ansible_sdk/model/job_status.py

```diff
@@ -79,8 +79,8 @@ async def stdout_lines(self) -> t.AsyncIterator[str]:
         """
         async for ev in self.events:
             # normalize some runner oddities
-            if 'stdout' in ev:
-                new_data = ev['stdout']
+            if ev.stdout:
```

Comment:
When I went looking at this, the actual data structure you have is a `dict`, so the code works, but `ev` is mis-typed. So I believe this is wrong now.

Reply:
ev is a dataclass obj `AnsibleJobStatusEvent`. There is no advantage of having it as `dict`. We will stick to dataclass and modify the code accordingly. Is that OK with you?

Reply:
I'm saying that it already is a `dict`. `AnsibleJobExecutorBase._stream_events()` is adding a dict object to the events array (the return from `json.loads()`).

Reply:
Yeah, that's a bug from the back-and-forth thing on the event type a couple months back- I'll fix that today. We definitely want the SDK event type to be a custom type, but I'm OK with exposing the raw event data through a `Mapping` impl on it, which should also have the nice side-effect of not breaking any code folks may have written against the existing event objects, since it's still exposing the raw event data through a dict-ish indexer on the object.

Reply:
https://github.com/ansible/ansible-sdk/pull/83